config/genesis_mainnet.yaml

```diff
@@ -0,0 +1,98 @@
+account:
+  initBalances:
```

Comment:
Why do we need this for genesis_mainnet.yaml ?

Reply:
this file is the exact config we used to bootstrap the mainnet, it is same as https://github.com/iotexproject/iotex-bootstrap/blob/master/genesis_mainnet.yaml

it is only used to create the genesis block at bootstrap time, does not affect later sync and chain operation afterwards